dotcom-rendering/src/web/components/RichLink.tsx

```diff
@@ -191,8 +191,8 @@ const readMoreText: (contentType: string) => string = (contentType) => {
 };
 
 const getMainContributor: (tags: TagType[]) => string = (tags) => {
-	const contributorTags = tags.filter((t) => t.type === 'Contributor');
-	return contributorTags.length > 0 ? contributorTags[0].title : '';
+	const [firstContributorTag] = tags.filter((t) => t.type === 'Contributor');
+	return firstContributorTag ? firstContributorTag.title : '';
```

Comment:
should this use `getSoleContributor`?

Reply:
I think probably not? this function is called `getMainContributor` which implies it should return something, even if there are multiple contributors, whereas `getSoleContributor` looks like it only returns something if theres only ONE contributor.